src/pages/community/event-guidance.md

```diff
@@ -0,0 +1,173 @@
+
+* [Organizers](#guidance-for-organizers)
+* [Speakers](#guidance-for-speakers)
+
+## Guidance for organizers
+
+### ASF rules take precedence
+
+Review the ASF's [trademark policy](https://www.apache.org/foundation/marks/) to ensure compliance. Here are some important takeaways:
+
+* Clearly indicate that your event is community-driven and not an official ASF event.
+* Seek approval when the event name or primary branding features [Apache Druid](https://www.apache.org/foundation/marks/events#events).
+* [Refer to Apache marks, projects, and products](https://www.apache.org/foundation/marks/events#naming) appropriately in materials, and include a clear attribution statement.
+* Only ever use the Apache Druid logo to [refer to Apache Druid itself](https://www.apache.org/foundation/marks/events#graphics).
+* Always check for [conflicts](https://www.apache.org/foundation/marks/events#dates) with other Apache events.
+
+### Reach out to the PMC
+
+It’s a standard practice for organizers of Apache-related events to connect with the project PMCs.
+
+* Reach out to the PMC using teh Apache Druid dev-list.
+
+* Work with a local PMC member who can [represent the Apache Druid project](https://www.apache.org/foundation/marks/events#selection) (not their employer), making sure that your event aligns with the project’s mission and policies.
+
+### Think outside the Druid box
+
+As an organizer of an Apache Druid meetup, you'll be playing a part of a bigger movement that's full of cool and superb people! Take note of the ecosystem around Apache Druid - you'll be fostering connections across the technoscape!
+
+* [Apache Airflow](https://airflow.apache.org/community/)
+* [Apache Beam](https://beam.apache.org/community/)
+* [Apache Calcite](https://calcite.apache.org/community/)
+* [Apache Datasketches](https://datasketches.apache.org/docs/Community)
+* [Apache Flink](https://flink.apache.org/what-is-flink/community/)
+* [Apache Iceberg](https://iceberg.apache.org/community/)
+* [Apache Kafka](https://kafka.apache.org/contact)
+* [Apache NiFi](https://nifi.apache.org/community/)
+* [Apache Presto](https://prestodb.io/community/)
+* [Apache Pulsar](https://pulsar.apache.org/community/)
+* [Apache Spark](https://spark.apache.org/community.html)
+* [Apache Superset](https://superset.apache.org/community/)
+* [Apache Zookeeper](https://zookeeper.apache.org/)
+
+Remember that the focus of the event remains Apache Druid, so guide speakers to talk about how their technology relates to and works with Apache Druid.
+
+### Educate, educate, educate
+
+Lean into [providing education](https://www.apache.org/foundation/marks/events#education) about Apache Druid, rather than filling your diary with cool presentations about use cases.
```

Comment:
Maybe phrase this more like "cool presentations about use cases are great, but the main focus should be education".  Use cases do have value.